File: test/e2e/test-files/workspaces/quarto_inline_output/multiline_statement.py
# This is a multi-line statement in Python.
sum(
    v ** 0.5
    for v in [1, 2, 3]
)

# And here is another.
([1, 2, 3] +
    [4, 5, 6] +
    [7, 8, 9])

# And one more for good measure.
import pandas as pd
(pd.DataFrame(
    {"x": [1, 2, 3],
     "y": [4, 5, 6]})
    .query("x > 1")
    .assign(z=lambda d: d.x + d.y)
    .head())

File: test/e2e/test-files/workspaces/quarto_inline_output/multiline_statement.r
# This is a multi-line statement in R.
c(-1, -2, -3) |>
  abs() |>
  sqrt() |>
  sum()

# And here is another.
c(1, 2, 3) +
  c(4, 5, 6) +
  c(7, 8, 9)

# And one more for good measure.
data.frame(
  x = c(1, 2, 3),
  y = c(4, 5, 6)
) |>
  subset(x > 1) |>
  transform(z = x + y) |>
  head()
